### PR TITLE
Update project for latest GWT, poms for release

### PR DESCRIPTION
**Pull request by niloc132**

### CODE REVIEW

File: gwt-json/pom.xml [hunk 1]
```diff
@@ -2,57 +2,23 @@
...
+        <artifactId>gwt-json-parent</artifactId>
+        <version>dev</version>
+    </parent>
     <artifactId>gwt-json</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>${revision}</version>
     <packaging>gwt-lib</packaging>
 
     <name>GWT JSON</name>
     <description>TBD</description>
```
Comment (niloc132) [inline gwt-json/pom.xml:15]: needs a description

**Comment by niloc132** [comment]
Need to update .gitignore to include `.flattened-pom.xml` - Frank can you update that to the template you are using when you do readme?

**Review by FrankHossfeld — APPROVED** [review]

File: gwt-json/pom.xml (hunk 1 above)
Comment (FrankHossfeld) [reply gwt-json/pom.xml:15]: see my PR

**Comment by FrankHossfeld** [comment]
See my PR ...

**Review by FrankHossfeld — APPROVED** [review]

**Pull request merged**